frontend/src/app/main/data/workspace/libraries_helpers.cljs

```diff
@@ -316,19 +316,21 @@
             :library (pretty-file library-id state))
 
   (let [file          (wsh/get-file state file-id)
-        components-v2 (get-in file [:options :components-v2])]
-    (loop [pages (vals (get file :pages-index))
+        components-v2 (get-in file [:options :components-v2])
+        deleted-components (ctkl/deleted-components-seq file)]
+    ;; We want to update the items inside the :objects of the deleted components. Those aren't in any page
+    (loop [items (concat (vals (get file :pages-index)) deleted-components)
```

Comment:
you could also replace this by the helper `ctpl/pages-seq`

Reply:
even more: perhaps you can create a helper in `app.common.file` called `object-containers-seq`, next to `containers-seq`, that gives you that: all containers in a file that has `:objects`